SwiftWisdom/Core/UIKit/UILabel+Typography.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  UILabel+Typography.swift
+//  SwiftWisdom
+//
+//  Created by Patrick Butkiewicz on 4/15/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+extension UILabel {
+    
+    /**
+     Sets the text of the UILabel instance to an attributed string with tracking applied
+     - Parameter value: The value in points that each character is kerned to
+     */
+    
+    public func ip_setCharacterSpacing(value: CGFloat) {
+        addAttribute(NSKernAttributeName, value: value)
+    }
+    
+    /**
+     Sets the text of the UILabel instance to an attributed string with the specified line spacing
+     - Parameter value: The space, in points, between each line.
+     */
+    
+    public func ip_setLineSpacing(value: CGFloat) {
+        let paragraphSpacing = NSMutableParagraphStyle()
+        paragraphSpacing.lineSpacing = value
+        addAttribute(NSParagraphStyleAttributeName, value: paragraphSpacing)
```

Comment:
This could override some existing paragraph style? Can you try to grab that and edit it before assigning a new one?